src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseInstanceFailoverGroup.md

```diff
@@ -0,0 +1,109 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlDatabaseInstanceFailoverGroup
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@jugeorge please make sure that you fill out all of the template sections in this file (synopsis, description, examples)